jobtracker.go

```diff
@@ -42,8 +42,12 @@ const (
 )
 
 var countersToKeep = map[string]string{
-	"FileSystemCounter.HDFS_BYTES_READ":    "hdfs.bytes_read",
-	"FileSystemCounter.HDFS_BYTES_WRITTEN": "hdfs.bytes_written",
+	"FileSystemCounter.HDFS_BYTES_READ":    "bytes_read",
+	"FileSystemCounter.S3_BYTES_READ":      "bytes_read",
+	"FileSystemCounter.FILE_BYTES_READ":	"bytes_read",
+	"FileSystemCounter.HDFS_BYTES_WRITTEN": "bytes_written",
+	"FileSystemCounter.S3_BYTES_WRITTEN":   "bytes_written",
+	"FileSystemCounter.FILE_BYTES_WRITTEN": "bytes_written",
```

Comment:
I think it would be preferable to send these as `(hdfs|s3|file).bytes_(read|written)` and let the summing happen on the client. That would let us split them into the separate parts if we wanted to show that. (I also have a vague memory of not wanting to pay attention to the FILE info for some reason).